src/components/Button.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { pallete } from "../misc/pallete";
+
+export default function Button({ size, text, onClick, clicked = false }) {
+  return size === "small" ? (
```

Comment:
tutaj można pójść o krok dalej - w TypeScript skorzystalibyśmy z typu 'enum' (na pewno dojdziemy do tego), ale w przypadku czystego JSa warto stworzyć obiekt, który będzie zawierał w sobie możliwe warianty właściwości size. 
Czyli np:
```
const buttonSize={
small:'small',
medium:'medium'
}
```
Co nam to daje? Przede wszystkim unikamy wszelkich literówek - bazujemy na właściwości obiektu poprzez notację 'buttonSize.small' dzięki czemu nie musimy tej właściwości zapisywać jako string. Mamy też wtedy pewność, że wartość której używamy jest jednakowa w całej aplikacji i mamy jedno 'źródło prawdy' - miejsce, w którym mamy zdefiniowane jakieś wartości i z tych wartości korzystamy w całej apce